packages/magika.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,4 +4,6 @@ Import-Module vm.common -Force -DisableNameChecking
 $toolName = 'magika'
 $category = 'File Information'
 
-VM-Install-With-Pip -toolName $toolName -category $category
+VM-Pip-Install 'magika==0.5.0'
+$executablePath = "$(where.exe $toolName)"
+VM-Install-Shortcut $toolName $category $executablePath -consoleApp $true -arguments $arguments
```

Comment:
I would like that we continue to use `VM-Install-With-Pip` by extending it to add a version argument. As already mentioned in other recent issues (for example https://github.com/mandiant/VM-Packages/issues/1080#issuecomment-2523816211), we should start fixing the version of the tools installed with Pip. So this is a common need and not something particular of this package. Extending the helper simplifies the code, remove code duplication and ensure consistency around different packages.

@emtuls  could you please extend the helper so that this package looks like:
```suggestion
$version = "==0.5.0"

VM-Install-With-Pip -toolName $toolName -category $category -version $version
```